db/external_sst_file_ingestion_job.cc

```diff
@@ -389,14 +389,25 @@ Status ExternalSstFileIngestionJob::IngestedFilesOverlapWithMemtables(
 Status ExternalSstFileIngestionJob::AssignLevelForIngestedFile(
     SuperVersion* sv, IngestedFileInfo* file_to_ingest, bool* overlap_with_db) {
   *overlap_with_db = false;
-
   Arena arena;
   ReadOptions ro;
   ro.total_order_seek = true;
 
   Status status;
   int target_level = 0;
   auto* vstorage = cfd_->current()->storage_info();
+
+  // Always go to last level, fail if it's not empty
+  if (ingestion_options_.ingest_behind) {
+    int lvl = cfd_->NumberLevels() - 1;
+    if (vstorage->NumLevelFiles(lvl) > 0) {
```

Comment:
This means that we can ingest one and only one file, I think we can allow ingesting in the bottommost level as long as our file don't overlap with any other files in the bottommost level